poh/src/poh_recorder.rs

```diff
@@ -520,6 +527,21 @@ impl PohRecorder {
                         target_tick_elapsed.as_micros(),
                         i64
                     ),
+                    (
+                        "was_reset_to_run_immediately",
+                        self.was_reset_to_run_immediately(leader_tick_heights.first_tick_height),
+                        bool
+                    ),
+                    (
+                        "prev_slot_was_mine",
+                        self.prev_slot_was_mine(new_leader_slot),
+                        bool
+                    ),
+                    (
+                        "is_same_fork_as_previous_leader",
+                        self.is_same_fork_as_previous_leader(new_leader_slot),
+                        bool
+                    ),
```

Comment:
@carllin I added this extra context to the metrics, is this what you were thinking?